### PR TITLE
[Merged by Bors] - feat(StrongTopology): generalize `ContinuousLinearMap.restrictScalarsL`

### PR DESCRIPTION
**Pull request by urkud**
---
- [x] depends on: #15289

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/LocallyConvex/Basic.lean [hunk 1]
```diff
@@ -250,7 +250,7 @@ end NormedField
 
 section NontriviallyNormedField
 
-variable [NontriviallyNormedField 𝕜] [AddCommGroup E] [Module 𝕜 E] {s : Set E}
+variable [NontriviallyNormedField 𝕜] [AddCommGroup E] [Module 𝕜 E] {s t : Set E}
```
Comment (urkud) [inline Mathlib/Analysis/LocallyConvex/Basic.lean:253]:
```suggestion
variable [NontriviallyNormedField 𝕜] [AddCommGroup E] [Module 𝕜 E] {s : Set E}
```

**Comment by urkud** [comment]
@j-loreaux You've marked this PR as `awaiting-author` but didn't leave any comments.

**Comment by urkud** [comment]
@j-loreaux I implemented the requested changes.

**Comment by j-loreaux** [comment]
bors merge

**Pull request closed without merging**